packages/frontend/core/src/components/pure/cmdk/main.tsx

```diff
@@ -78,11 +79,15 @@ const QuickSearchGroup = ({
                 title: command.label,
               }
             : command.label;
+
+        // use to remove double quotes from a string until this issue is fixed
+        // https://github.com/pacocoursey/cmdk/issues/189
+        const escapeValue = removeDoubleQuotes(command.value);
```

Comment:
As I said, you should escape the value in `pageToCommand`

Reply:
The implementation of create page as xxx is not based on pageToCommand, and it seems that there is no guarantee that other commands will not have similar problems in the future.

Reply:
hmm, sounds good.